invesalius_cy/mask_cut.pyx

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+
+from libc.math cimport floor, ceil, sqrt, fabs, round
+from cython.parallel import prange
+
+DTYPEF64 = np.float64
+ctypedef np.float64_t DTYPEF64_t
+
+ctypedef fused image_t:
+    np.float64_t
+    np.int16_t
+    np.uint8_t
+
+ctypedef np.uint8_t mask_t
```

Comment:
You can import the types from cy_my_types https://github.com/invesalius/invesalius3/blob/master/invesalius_cy/cy_my_types.pxd. Like here https://github.com/invesalius/invesalius3/blob/master/invesalius_cy/floodfill.pyx#L21

Reply:
Thank you, I have solved that. Please check the new code